### DISCUSSION TITLE
Is the prisma-cli binary a native rust binary or a wrapper around node prisma?

### DISCUSSION THREAD

**Discussion by natevaughan** [Brendonovich/prisma-client-rust#443] [Q&A] [answered]
Hi, thanks for your fantastic project. A colleague and I are trying to resolve a disagreement. I've installed the prisma-cli as a cargo workspace.

When I invoke e.g. `cargo prisma migrate dev`, is the underlying binary 100% rust binary or a wrapper around node prisma?

**Comment by natevaughan** [comment]
We think we figured it out. Looks like you maintain a fork of the underlying (official Prisma) rust `prisma-engines` project to control versioning, and then specify and invoke those forked engines. Do we have that right?

**Reply by Brendonovich** [reply, marked as answer]
The engines fork is for the runtime crate, the CLI crate is a wrapper around Prisma's precompiled CLI
